consensus/hotstuff/votecollector/collection_cluster_vote_collector.go

```diff
@@ -0,0 +1,28 @@
+package votecollector
+
+import (
+	"go.uber.org/atomic"
+
+	"github.com/onflow/flow-go/consensus/hotstuff"
+	"github.com/onflow/flow-go/consensus/hotstuff/model"
+)
+
+type CollectionClusterVoteCollector struct {
```

Comment:
See my comment on `Cluster` above.

Besides this, how do we name / comment this so that it's very clear that `Collection` refers to a node role, `Collector` refers to vote storage and processing, and that those are absolutely not the same?

Reply:
Basically I have used word `Cluster` to make it specific that `Collection` and `Consensus` refer to node types. This way `Collector` means that it's collecting something. I can change `CollectionClusterVoteCollector` -> `CollectionVoteCollector` dunno if that's better